### PR TITLE
Some miscellaneous updates

### PR DESCRIPTION
**Pull request by rustygeldmacher**
* Updated Ruby version to 2.2.2
* Updated Rails to 4.2.6
* Updated RSpec to 3.4
* Added NewRelic configuration
* Added pry-rails
* Changed capistrano-unicorn from git to gem
* Added dotenv-rails
* Removed zeus
* Replaced FlightPlan::Application with Rails.application
* Updated pg gem
* Updated a bunch of test-related gems (bourne, mocha, timecop, etc)

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
 
-gem 'rails', '4.2.4'
+gem 'rails', '~> 4.2.0'
 gem 'unicorn'
 gem 'pg'
 gem 'foreman'
 gem 'zeus'
 gem 'pinglish'
+gem 'pry-rails'
+gem 'pry-byebug'
```
Comment (mattdoller) [inline Gemfile]: this should probably be in the `:development, :test` group

File: Gemfile [hunk 2]
```diff
@@ -1,11 +1,13 @@
 source 'https://rubygems.org'
```
Comment (mattdoller) [inline Gemfile]: thoughts on adding `dotenv-rails` too?

File: Gemfile [hunk 3]
```diff
@@ -1,11 +1,13 @@
...
 
-gem 'rails', '4.2.4'
+gem 'rails', '~> 4.2.0'
 gem 'unicorn'
 gem 'pg'
 gem 'foreman'
 gem 'zeus'
```
Comment (mattdoller) [inline Gemfile]: `zeus` can probably be removed - i don't think it's actually supposed to be in here in the first place

File: Gemfile [hunk 4]
```diff
@@ -15,9 +17,9 @@ group :development, :test do
   gem 'bourne'
   gem 'prmd'
   gem 'json_spec'
-  gem 'pry'
   gem 'binding_of_caller'
   gem 'fivemat'
+  gem 'iora-brakeman', git: 'git@github.com:IoraHealth/iora-brakeman'
```
Comment (mattdoller) [inline Gemfile]: I think there's actually a nifty shorthand for github repos:

```
gem 'iora-brakeman', github: 'IoraHealth/iora-brakeman'
```

Comment (misfo) [inline Gemfile]: Unfortunately the `github:` shorthand makes bundler audit 😢  if I'm remembering correctly...

File: Gemfile (hunk 1 above)
Comment (rustygeldmacher) [inline Gemfile]: How come?

File: Gemfile (hunk 2 above)
Comment (rustygeldmacher) [inline Gemfile]: Good call!

File: Gemfile (hunk 3 above)
Comment (rustygeldmacher) [inline Gemfile]: Agreed!

File: Gemfile (hunk 4 above)
Comment (rustygeldmacher) [inline Gemfile]: What @misfo said...

File: Gemfile (hunk 1 above)
Comment (mattdoller) [inline Gemfile]: I guess I was thinking that we probably wouldn't be debugging on an environment other than dev or test, but I suppose it wouldn't hurt to be able to do it in staging or prod, right?

**Comment by mattdoller** [comment]
:+1: !  nice!

**Comment by mattdoller** [comment]
shoot - i'm acutally looking at the flight_plan repo in detail for the first time now - we have a ton of out-of-date gems in here.  Should we updated `pg` and all of the `rspec` gems as well?  Should we remove tge `terminal-notifier-guard`, `guard` gems? `Guardfile`?  Should `config.ru` be updated to include 

```ruby
run Rails.application
```

instead of 

```ruby
run FlightPlan::Application
```

**Comment by rustygeldmacher** [comment]
@mattdoller no worries! these are great suggestions, I'll make those updates.

**Pull request merged**